shared/profile/wot-author/index.tsx

```diff
@@ -35,27 +53,78 @@ const WotModal = (props: WotModalProps) => {
       footer={{
         content: (
           <Kb.ButtonBar align="center" direction="row" fullWidth={true} style={styles.buttonBar}>
-            <Kb.Button fullWidth={true} label={props.submitLabel} onClick={props.onSubmit} />
+            <Kb.Button
+              fullWidth={true}
+              label={props.submitLabel}
+              onClick={props.onSubmit}
+              disabled={props.submitDisabled}
+              waiting={props.submitWaiting}
+            />
           </Kb.ButtonBar>
         ),
       }}
     >
-      <Kb.Box2
-        direction="vertical"
-        alignSelf="stretch"
-        alignItems="stretch"
-        style={{backgroundColor: Styles.globalColors.green}}
-      >
+      <Kb.Box2 direction="vertical" alignSelf="stretch" alignItems="center" style={styles.topIconContainer}>
         <Kb.Icon type="icon-illustration-wot-460-96" />
       </Kb.Box2>
       {props.children}
     </Kb.Modal>
   )
 }
 
-export const Question1 = (props: Props) => {
+type Question1Props = {
+  error?: string
+  initialVerificationType?: WebOfTrustVerificationType
+  onSubmit: (_: {
+    otherText: string
+    proofs: {key: string; value: string}[]
+    verificationType: WebOfTrustVerificationType
+  }) => void
+  proofs: {key: string; value: string}[]
+  voucheeUsername: string
+}
+
+export const Question1 = (props: Question1Props) => {
+  const scrollViewRef = React.useRef<Kb.ScrollView>(null)
+  const [vt, setVtRaw] = React.useState<WebOfTrustVerificationType>(
```

Comment:
`setVtRaw` might be called `_setVt`. Also, I find `vt` to be a mildly inscrutable name if you have any better ideas that aren't interminably long.